en/authentication_authorization/README.md

```diff
@@ -144,7 +144,7 @@ Let's add some sugar to our templates while we're at it. First we will add some
 
 This adds a nice "Hello _&lt;username&gt;_" to remind us who we are logged in as, and that we are authenticated. Also, this adds a link to log out of the blog -- but as you might notice this isn't working yet. Let's fix it!
 
-We decided to rely on Django to handle login, so let's see if Django can also handle logout for us. Check https://docs.djangoproject.com/en/1.10/topics/auth/default/ and see if you find something.
+We decided to rely on Django to handle login, so let's see if Django can also handle logout for us. Check https://docs.djangoproject.com/en/2.1/topics/auth/default/ and see if you find something.
```

Comment:
As long as the Django Girls tutorial [is using Django 2.0](https://github.com/DjangoGirls/tutorial/blob/c68164c9486d70483b3f9e3d62b8129185b5700d/book.json#L4), the extensions should probably also link to the 2.0 documentation, not the Django 2.1 one.

Reply:
@das-g Thank you for review.
I agree that the extensions should link to the Django 2.0 documentation (same version as Django Girls Tutorial).
Fixed at https://github.com/DjangoGirls/tutorial-extensions/pull/121/commits/5a2dec22ecab6ff845d716ad8ab29917adaf8664.